wpengine/unibody
# [WEB-3433] Update list type overrides for ordered lists

NOTE - alternate solution that retains existing unibody styles here:
https://github.com/wpengine/wpengine-breakthrough/pull/191

## JIRA Ticket
[A link to the relevant JIRA ticket](https://wpengine.atlassian.net/browse/WEB-3433)

## Description
A few sentences describing the overall goals of the pull request's commits.

- Ordered lists are not respecting "start" attribute. 
- Ordered lists no longer require special styling, will follow default styling moving forward (1., 2., etc)
- Fixes build process error

## Impacted Areas in Application
List general components of the site that this PR will affect:

* all uses of Ordered Lists

## Deploy Notes
Notes regarding deployment the contained body of work.
If you need to change something in WordPress after deploying, make the necessary steps crystal clear.

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. link unibody to wpengine
2. Test on page with ordered links using start attribute. Example: wpengine.local/support/git 
3. Ordered lists should respect "start attribute"
4. Ordered lists should display in the default mode of 1., 2., 3., etc.

NOTE - I was not able to view these changes locally, so going on faith a bit here. I have a workaround that will involve overriding unibody on Breakthrough, but updating unibody is the desired approach. 


## Random GIF
Include one random GIF to "pay" your reviewers in humor.

![Thanks for reviewing my code!](https://media.giphy.com/media/UaJAJPoSf1KYo/giphy.gif)
